<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="DESKTOP-8MO52H1" failures="15" tests="17" name="Command line test" time="5.281" errors="0" timestamp="٢٠٢١-٠١-٢١T٠٩:٣٨:٢١ EET">
  <testcase classname="TestRespones" name="testDelAllEmp" time="0.188">
    <failure type="java.lang.AssertionError" message="expected [200] but found [415]">
      <![CDATA[java.lang.AssertionError: expected [200] but found [415]
at TestRespones.testDelAllEmp(TestRespones.java:118)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 43 stack frames]]>
    </failure>
  </testcase> <!-- testDelAllEmp -->
  <testcase classname="TestRespones" name="testDelEmp" time="0.164">
    <failure type="java.lang.AssertionError" message="expected [200] but found [415]">
      <![CDATA[java.lang.AssertionError: expected [200] but found [415]
at TestRespones.testDelEmp(TestRespones.java:131)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 43 stack frames]]>
    </failure>
  </testcase> <!-- testDelEmp -->
  <testcase classname="TestRespones" name="testDelEmp" time="0.155">
    <failure type="java.lang.AssertionError" message="expected [200] but found [415]">
      <![CDATA[java.lang.AssertionError: expected [200] but found [415]
at TestRespones.testDelEmp(TestRespones.java:131)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 43 stack frames]]>
    </failure>
  </testcase> <!-- testDelEmp -->
  <testcase classname="TestRespones" name="testDelEmp" time="0.178">
    <failure type="java.lang.AssertionError" message="expected [200] but found [415]">
      <![CDATA[java.lang.AssertionError: expected [200] but found [415]
at TestRespones.testDelEmp(TestRespones.java:131)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 43 stack frames]]>
    </failure>
  </testcase> <!-- testDelEmp -->
  <testcase classname="TestRespones" name="testDelEmp" time="0.177">
    <failure type="java.lang.AssertionError" message="expected [200] but found [415]">
      <![CDATA[java.lang.AssertionError: expected [200] but found [415]
at TestRespones.testDelEmp(TestRespones.java:131)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 43 stack frames]]>
    </failure>
  </testcase> <!-- testDelEmp -->
  <testcase classname="TestRespones" name="testDelEmp" time="0.16">
    <failure type="java.lang.AssertionError" message="expected [200] but found [415]">
      <![CDATA[java.lang.AssertionError: expected [200] but found [415]
at TestRespones.testDelEmp(TestRespones.java:131)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 43 stack frames]]>
    </failure>
  </testcase> <!-- testDelEmp -->
  <testcase classname="TestRespones" name="testGetAllEmp" time="0.186"/>
  <testcase classname="TestRespones" name="testMaxMinSal" time="0.269"/>
  <testcase classname="TestRespones" name="testPostEmp" time="0.201">
    <failure type="java.lang.AssertionError" message="expected [0] but found [-14]">
      <![CDATA[java.lang.AssertionError: expected [0] but found [-14]
at TestRespones.testPostEmp(TestRespones.java:60)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 43 stack frames]]>
    </failure>
  </testcase> <!-- testPostEmp -->
  <testcase classname="TestRespones" name="testPostEmp" time="0.155">
    <failure type="java.lang.AssertionError" message="expected [0] but found [-14]">
      <![CDATA[java.lang.AssertionError: expected [0] but found [-14]
at TestRespones.testPostEmp(TestRespones.java:60)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 43 stack frames]]>
    </failure>
  </testcase> <!-- testPostEmp -->
  <testcase classname="TestRespones" name="testPostEmp" time="0.2">
    <failure type="java.lang.AssertionError" message="expected [0] but found [-14]">
      <![CDATA[java.lang.AssertionError: expected [0] but found [-14]
at TestRespones.testPostEmp(TestRespones.java:60)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 43 stack frames]]>
    </failure>
  </testcase> <!-- testPostEmp -->
  <testcase classname="TestRespones" name="testPostEmp" time="0.199">
    <failure type="java.lang.AssertionError" message="expected [0] but found [-14]">
      <![CDATA[java.lang.AssertionError: expected [0] but found [-14]
at TestRespones.testPostEmp(TestRespones.java:60)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 43 stack frames]]>
    </failure>
  </testcase> <!-- testPostEmp -->
  <testcase classname="TestRespones" name="testPostEmp" time="0.166">
    <failure type="java.lang.AssertionError" message="expected [0] but found [-14]">
      <![CDATA[java.lang.AssertionError: expected [0] but found [-14]
at TestRespones.testPostEmp(TestRespones.java:60)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 43 stack frames]]>
    </failure>
  </testcase> <!-- testPostEmp -->
  <testcase classname="TestRespones" name="testPutEmp" time="0.402">
    <failure type="java.lang.AssertionError" message="expected [2500] but found [1250.0]">
      <![CDATA[java.lang.AssertionError: expected [2500] but found [1250.0]
at TestRespones.testPutEmp(TestRespones.java:103)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 42 stack frames]]>
    </failure>
  </testcase> <!-- testPutEmp -->
  <testcase classname="TestRespones" name="testPutEmp" time="0.373">
    <failure type="java.lang.AssertionError" message="expected [Qasem] but found [Osama]">
      <![CDATA[java.lang.AssertionError: expected [Qasem] but found [Osama]
at TestRespones.testPutEmp(TestRespones.java:101)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 42 stack frames]]>
    </failure>
  </testcase> <!-- testPutEmp -->
  <testcase classname="TestRespones" name="testPutEmp" time="0.408">
    <failure type="java.lang.AssertionError" message="expected [Raed] but found [Thaer]">
      <![CDATA[java.lang.AssertionError: expected [Raed] but found [Thaer]
at TestRespones.testPutEmp(TestRespones.java:101)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 42 stack frames]]>
    </failure>
  </testcase> <!-- testPutEmp -->
  <testcase classname="TestRespones" name="testPutEmp" time="0.425">
    <failure type="java.lang.AssertionError" message="expected [Rand] but found [Wael]">
      <![CDATA[java.lang.AssertionError: expected [Rand] but found [Wael]
at TestRespones.testPutEmp(TestRespones.java:101)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 42 stack frames]]>
    </failure>
  </testcase> <!-- testPutEmp -->
</testsuite> <!-- Command line test -->
